<launch>
    <param name="/use_sim_time" value="false"/>

    <arg name="bag_name1" value="tmp_a_acbf.bag"/>
    <arg name="bag_name2" value="tmp_2.bag"/>

    <!-- topic /odom_traj /robot1/odom /corridor_vis -->
    <node pkg="rosbag" type="play" name="player1" args="--clock /home/pj/jane_ws-github/dynamic_avoidance/bag/0725/gazebo-s1/$(arg bag_name1) " >
        <remap from="/odom_traj" to="/odom_traj1"/>
        <remap from="/robot1/odom" to="/odometry1"/>
    </node>

    <node pkg="robot_simulator" type="scout_visualizer_node" name="scout_visualizer_node1" output="screen">
        <param name="ugv/mesh" value="package://robot_simulator/param/scout2.STL" type="string"/>
        <remap from="~odometry" to="/odometry1"/>
        <remap from="~odom_mesh" to="robot_mesh1"/>
        <param name="ugv/color_r" value="0.0" type="double"/>
        <param name="ugv/color_g" value="0.0" type="double"/>
        <param name="ugv/color_b" value="1.0" type="double"/>
        <param name="ugv/color_a" value="0.3" type="double"/>
    </node>

    <node pkg="rosbag" type="play" name="player2" args="--clock /home/pj/jane_ws-github/dynamic_avoidance/bag/0725/gazebo-s1/$(arg bag_name2) --topic /odom_traj /robot1/odom" >
        <remap from="/odom_traj" to="/odom_traj2"/>
        <remap from="/robot1/odom" to="/odometry2"/>
    </node>

    <node pkg="robot_simulator" type="scout_visualizer_node" name="scout_visualizer_node2" output="screen">
        <param name="ugv/mesh" value="package://robot_simulator/param/scout2.STL" type="string"/>
        <remap from="~odometry" to="/odometry2"/>
        <remap from="~odom_mesh" to="robot_mesh2"/>
        <param name="ugv/color_r" value="0.0" type="double"/>
        <param name="ugv/color_g" value="1.0" type="double"/>
        <param name="ugv/color_b" value="0.0" type="double"/>
        <param name="ugv/color_a" value="0.3" type="double"/>
    </node>

    <node pkg="rviz" type="rviz" name="rviz" output="log" args="-d $(find swarm_test)/config/show_mpc_traj.rviz" />
</launch>